src/renderer/api/comms.ts

```diff
@@ -1,3 +1,18 @@
+/**
+ * @fileoverview Comms API
+ *
+ * This documentation comment created on 12APR2023 by @knoxy5467
+ *
+ * last updated on 12APR2023 by @knoxy5467
+ *
+ * @class CommsAPI
+ * @classdesc Comms API
```

Comment:
Can you remove this fluff? I'm keen on adding more overview documentation to classes, but I'd like to keep it to only the bare necessitities. If we ever get into auto-generating frontend documentation based on the code comments, a lot should be able to be inferred from Typescript.

Looking as JSDoc, I think this should probably be:
```
@fileoverview This class is a extension of the TachyonClient class. It is used to communicate with the server. All communication with the server should be done through this class, includeing matchmaking, chat, direct messages, and other lobby related functions.
```